packages/account/integration-tests/personal-details/personal-details.spec.tsx

```diff
@@ -74,8 +88,25 @@ test.describe('Personal Details', () => {
 
         const submitButton = await page.getByRole('button', { name: 'Submit' }).first();
         expect(await submitButton.isEnabled()).toBe(true);
-        submitButton.click();
+        await submitButton.click();
 
         await page.waitForSelector('[data-testid=form-footer-container] button svg');
     });
+
+    test('should show error message when submitting form with invalid phone number', async ({ page, baseURL }) => {
+        await setupMocks({
+            baseURL,
+            page,
+            mocks: [mock_general, mock_loggedIn, mock_residents_list, mock_states_list, mock_set_settings_invalid],
```

Comment:
```suggestion
            mocks: [mockGeneral, mockLoggedIn, mockResidentsList, mockStatesList, mockSetSettingsInvalid],
```
I noticed that all these mocks are functions. According to our naming convention, functions are named in camelCase :) same for the rest of the mocks

Reply:
Yeah great point Maryia. I was trying to be consistent with some of the old style guide, but since we're switching to a more pure Airbnb styleguide, these should all be camel case. Can we merge this for now, then I'll make another PR to refactor all these variables?

Reply:
sure, @markwylde-deriv , what about doing it in this new one https://github.com/binary-com/deriv-app/pull/11418/files ?  It's about refactoring :)